ui/src/wizard/AzureWizard.ts

```diff
@@ -20,14 +20,18 @@ export class AzureWizard<T> {
         this._wizardContext = wizardContext;
     }
 
-    public async prompt(actionContext: IActionContext, ui: IAzureUserInput): Promise<T> {
+    public async prompt(actionContext: IActionContext, ui: IAzureUserInput, parentWizard?: AzureWizard<T>): Promise<T> {
         for (const step of this._promptSteps) {
+            if (parentWizard && parentWizard._promptSteps.some((parentStep: AzureWizardPromptStep<T>) => parentStep.constructor.name === step.constructor.name)) {
```

Comment:
Might this need to traverse into the parent wizard's parent wizard's steps?

Reply:
Fixed